addons/website/models/ir_actions.py

```diff
@@ -60,18 +60,3 @@ def _run_action_code_multi(self, eval_context=None):
         """
         res = super(ServerAction, self)._run_action_code_multi(eval_context)
         return eval_context.get('response', res)
-
-
-class IrActionsTodo(models.Model):
-    _name = 'ir.actions.todo'
-    _inherit = 'ir.actions.todo'
-
-    def action_launch(self):
-        res = super().action_launch()  # do ensure_one()
-
-        if self.id == self.env.ref('website.theme_install_todo').id:
-            # Pick a theme consume all ir.actions.todo by default (due to lower sequence).
-            # Once done, we re-enable the main ir.act.todo: open_menu
-            self.env.ref('base.open_menu').action_open()
```

Comment:
why remove it ? 

When you install ecommerce, once you have choose your theme, you want to get back to the prvious open_menu action -> start tour or open ecommerce or ...